README.md

```diff
@@ -87,6 +87,7 @@ Other open source projects that use Argo:
 * [Seldon](https://github.com/SeldonIO/seldon-core) is an MLOps framework to package, deploy, monitor and manage thousands of production machine learning models.
 * [SQLFlow](https://github.com/sql-machine-learning/sqlflow) extends SQL to support AI and compiles the SQL program to a workflow that runs on Kubernetes.
 * [terraform-provider-argocd](https://github.com/oboukili/terraform-provider-argocd) is the Terraform provider for Argo CD.
+* [Flowify](https://equinor.github.io/flowify-documentation/) Collaborative no code UI for Argo Workflows.
```

Comment:
This is linking to the documentation?

Reply:
We have 3 separate repos for the project. The doc has link to them. Or you want to link to just backend or frontend?